portugues/colabora/comdiferencas/diff/pull/reenvio/watch/divulga/colaborador/colaborador.md

```diff
@@ -0,0 +1,2 @@
+Você continua atuando apenas como colaborador no projeto. Por causa da sua boa contribuição
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.